recipe/meta.yaml

```diff
@@ -9,7 +9,8 @@ source:
   sha256: d4fc65b98e348c39d082ab6b4b7f6d636b1b4e63bec016e5bca189fee5d46403
 
 build:
-  number: 0
+  skip: true  # [py37]
```

Comment:
Argh. I missed this :unamused: 

Can you please send another PR removing this and re-rendering?